packages/webpack/src/config/base.js

```diff
@@ -327,15 +327,27 @@ export default class WebpackBaseConfig {
   }
 
   plugins() {
-    const plugins = [new VueLoader.VueLoaderPlugin()]
-
-    Array.prototype.push.apply(plugins, this.options.build.plugins || [])
+    const plugins = []
 
     // Add timefix-plugin before others plugins
     if (this.options.dev) {
-      plugins.unshift(new TimeFixPlugin())
+      plugins.push(new TimeFixPlugin())
     }
 
+    // CSS extraction)
+    if (this.options.build.extractCSS) {
+      plugins.push(new ExtractCssChunksPlugin(Object.assign({
+        filename: this.getFileName('css'),
+        chunkFilename: this.getFileName('css'),
+        // TODO: https://github.com/faceyspacey/extract-css-chunks-webpack-plugin/issues/132
+        reloadAll: true
+      }, this.options.build.extractCSS)))
+    }
+
+    plugins.push(new VueLoader.VueLoaderPlugin())
+
+    Array.prototype.push.apply(plugins, this.options.build.plugins || [])
```

Comment:
@clarkdo do you know why we can't use `plugins.push` here?

Reply:
It was a legacy issue :laughing: I think it is now even safe to use rest operator.

Reply:
@manniL No special reason, it's old code, I didn't notice it